pkg/controller/snapshotter_test.go

```diff
@@ -50,331 +54,14 @@ func createMockServer(t *testing.T) (*gomock.Controller, *driver.MockCSIDriver,
 
 	// Create a client connection to it
 	addr := drv.Address()
-	csiConn, err := New(addr)
+	csiConn, err := connection.Connect(addr)
 	if err != nil {
 		return nil, nil, nil, nil, nil, err
 	}
 
 	return mockController, drv, identityServer, controllerServer, csiConn, nil
 }
 
-func TestGetPluginInfo(t *testing.T) {
-	tests := []struct {
-		name        string
-		output      *csi.GetPluginInfoResponse
-		injectError bool
-		expectError bool
-	}{
-		{
-			name: "success",
-			output: &csi.GetPluginInfoResponse{
-				Name:          "csi/example",
-				VendorVersion: "0.3.0",
-				Manifest: map[string]string{
-					"hello": "world",
-				},
-			},
-			expectError: false,
-		},
-		{
-			name:        "gRPC error",
-			output:      nil,
-			injectError: true,
-			expectError: true,
-		},
-		{
-			name: "empty name",
-			output: &csi.GetPluginInfoResponse{
-				Name: "",
-			},
-			expectError: true,
-		},
-	}
-
-	mockController, driver, identityServer, _, csiConn, err := createMockServer(t)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer mockController.Finish()
-	defer driver.Stop()
-	defer csiConn.Close()
-
-	for _, test := range tests {
-
-		in := &csi.GetPluginInfoRequest{}
-
-		out := test.output
-		var injectedErr error
-		if test.injectError {
-			injectedErr = fmt.Errorf("mock error")
-		}
-
-		// Setup expectation
-		identityServer.EXPECT().GetPluginInfo(gomock.Any(), in).Return(out, injectedErr).Times(1)
-
-		name, err := csiConn.GetDriverName(context.Background())
-		if test.expectError && err == nil {
-			t.Errorf("test %q: Expected error, got none", test.name)
-		}
-		if !test.expectError && err != nil {
-			t.Errorf("test %q: got error: %v", test.name, err)
-		}
-		if err == nil && name != "csi/example" {
-			t.Errorf("got unexpected name: %q", name)
-		}
-	}
-}
-
-func TestSupportsControllerCreateSnapshot(t *testing.T) {
```

Comment:
We still need a test for this.

Reply:
I still don't see tests for this?

Reply:
ah, sorry I thought this was list snapshot, fixing now